ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -284,6 +310,9 @@ const handleRasCallback = (code: string, spinnerProps: WithSpinnerOverlayProps,
     await profileApi().linkRasAccount({ authCode: code, redirectUrl: buildRasRedirectUrl() });
     spinnerProps.hideSpinner();
     reloadProfile();
+
+    // Cleanup parameter from URL after linking.
+    window.history.replaceState({}, '', '/');
```

Comment:
prevents the "RAS linkage" error when completing/refreshing ERA commons.  The cause of that error is because it tried to process the stale code from the URL again.  Now it does not do that.